performance-test/src/puppeteer-test.ts

```diff
@@ -127,7 +239,16 @@ async function createTestPng(
     percentile50: number | undefined
     percentile75: number | undefined
   },
-) {
+  isSelectionTest: boolean,
+): Promise<string | null> {
+  if (
+    process.env.PERFORMANCE_GRAPHS_PLOTLY_USERNAME == null ||
+    process.env.PERFORMANCE_GRAPHS_PLOTLY_API_KEY == null
+  ) {
+    console.info('Plotly graph generation skipped because of missing username or API key')
+    return null
+  }
+  const xAxisRange = isSelectionTest ? { min: 100, max: 200 } : { min: 0, max: 134 }
```

Comment:
I guess min and max range could be a parameter, and then this wouldn't be special cased to the selection test

Reply:
I just realised what this does. Not a fan of axis cutting in public, we should _always_ start at zero.

If we use the box plot, we'd automatically dimension the x-axis (showing the values) anyway, and there's an option in the graphing library to hide true outliers.

Reply:
I agree with starting from 0, I think there's a benefit to manually cutting on the 'right' side: it makes it easier to compare a test run with a one month old one, since you don't need to reinterpret what you see based on markings on the x axis